ros2_control_demo_hardware/include/ros2_control_demo_hardware/rrbot_system_position_only.hpp

```diff
@@ -34,7 +34,7 @@ class RRBotSystemPositionOnlyHardware
 : public hardware_interface::BaseInterface<hardware_interface::SystemInterface>
 {
 public:
-  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemPositionOnlyHardware);
+  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemPositionOnlyHardware)
```

Comment:
Revert this change because it fails on our liners.

```suggestion
  RCLCPP_SHARED_PTR_DEFINITIONS(RRBotSystemPositionOnlyHardware);
```